File: Irrgarten_Java/src/irrgarten/TestP3.java
package irrgarten;

import irrgarten.controller.Controller;
import irrgarten.UI.TextUI;

/**
 * Programa de prueba para la practica 3
 * 
 * @author Arturo Olivares Martos
 * @author Joaquín Avilés de la Fuente
 */
public class TestP3 {
    
    /**
     * Método que realiza las pruebas unia
     * tarias de la práctica 3
     * @param args Argumentos recibidos al ejecutar
     */
    public static void main ( String [ ] args ) {
        
        final int N_PLAYERS = 1;

        TextUI vista = new TextUI();
        Game juego = new Game(N_PLAYERS);
        Controller controlador = new Controller(juego, vista);

        controlador.play();
    }
}

File: Irrgarten_Ruby/test_p3.rb
# encoding: UTF-8

require_relative 'Controller/controller'
require_relative 'UI/textUI'
require_relative 'game'

module Irrgarten

    # Código para realizar las pruebas de la práctica 3
    #
    # @author Joaquin Avilés de la Fuente
    # @author Arturo Olivares Martos

    # Número de jugadores
    NUM_JUGADORES = 1

    vista=UI::TextUI.new  # Los dos puntos porque están en otro módulo
    juego=Game.new(NUM_JUGADORES)

    controller=Control::Controller.new(juego,vista)
    controller.play
end
